### ISSUE TITLE
Lifecycle history should have role attribute as log

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#9191]
### Description:

role=log
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA23.html
![image](https://user-images.githubusercontent.com/20179540/91313981-a9234c80-e7d3-11ea-80c5-482ce3ab53fa.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---

**Comment by erangatl** [comment]
since this is in publisher reducing the priority
